examples/container-deny-added-caps/template.yaml

```diff
@@ -28,12 +28,17 @@ spec:
           not is_gatekeeper
       }
 
-      format(msg) = msg {
-          not is_gatekeeper
+      format(msg, id) = msg_fmt {
```

Comment:
thought: I recall briefly talking about some `format()` history and why it was introduced. Before the days that Conftest support JSON in the result, and now wondered if this method was really needed! I'm doing some waffling because this will break consumers who do format(message), as ID is now required--even if they don't supply an ID.

So I feel like we either need to expose a couple different formats, or maybe just remove it to simplify the library and have the consumers handle it themselves?

Reply:
Good point. In the interest of not breaking existing flows, I've made a change where format() takes just a `msg` parameter and always uses the Gatekeeper format since Conftest supports that now, and have added a format_with_id() function.

Reply:
Lets rock and roll